# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/materialx/hotfix-cmake-4.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

CMake Error at /usr/src/t2-src/src.materialx.250717.190908.4235/MaterialX.git/source/MaterialXView/NanoGUI/resources/bin2c.cmak
e:1 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

--- MaterialX.git/source/MaterialXView/NanoGUI/resources/bin2c.cmake.vanilla	2025-07-17 19:10:16.913205395 +0300
+++ MaterialX.git/source/MaterialXView/NanoGUI/resources/bin2c.cmake	2025-07-17 19:10:25.081473703 +0300
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 2.8.12)
+cmake_minimum_required (VERSION 3.5.0)
 
 # Create header for C file
 file(WRITE ${OUTPUT_C} "/* Autogenerated by bin2c */\n\n")
